shumgrepper/api.py

```diff
@@ -78,29 +78,27 @@ def api_tar_sum(tar_sum):
     )
 
 
-# request files by tarsum
+# request files by tarball
 @app.route('/api/tarball/<tarball>/filenames')
-def api_tarball(tarball):
-    messages = sm.File.by_tarball(session, tarball)
-    file_list = []
-    for message in messages:
-        file_list.append(message.filename)
+def api_tarball_filenames(tarball):
+    filenames = sm.File.tarball_filenames(session, tarball)
+
+    filename_list = [filename[0][1:] for filename in filenames]
```

Comment:
But that's not where I commented (and that's why I added on this file: ``add the same comment here`` :-))